packages/core/src/components/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,82 @@
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Dropdown
+
+Dropdowns allow users to select one option from a temporary modal menu.
+
+@uswds https://designsystem.digital.gov/components/form-controls/#dropdown
+
+Markup: Dropdown.example.html
+
+Style guide: components.dropdown
+*/
+
+// stylelint-disable selector-no-qualifying-type
+select.ds-c-field {
+  $select-icon-size: 10px;
+  appearance: none;
+  background-color: $color-white;
+  background-image: url('#{$image-path}/arrow-both.svg');
+  background-position: right $spacer-1 center;
+  background-repeat: no-repeat;
+  background-size: $select-icon-size;
+  padding-right: ($select-icon-size * 2) + $spacer-1;
+
+  &[multiple] {
+    background-image: none;
+  }
+}
+// stylelint-enable selector-no-qualifying-type
+
+/*
+`<Dropdown>`
+
+@react-component Dropdown
+
+Style guide: components.dropdown.react
+*/
+
+/*
+---
+
+### When to use
+
+- **Use sparingly**. Use the dropdown component only when a user needs to choose from about seven to 15 possible options and you have limited space to display the options.
+
+### When to consider alternatives
+
+- **Fewer than seven options.** Use [radio buttons or checkboxes](/components/choice) instead.
+- **More than 15 options.** If the list of options is very long. Let users type the same information into a text input that suggests possible options instead.
+- **Multi-select.** If you need to allow users to select more than one option at once. Users often don’t understand how to select multiple items from dropdowns. Use [checkboxes](/components/choice) instead.
+- **Site navigation.** Use the [navigation components](/components/vertical-nav/) instead.
+
+## Guidance
+
+- **Make sure to test.** Test dropdowns thoroughly with members of your target audience. Several usability experts suggest they should be the “UI of last resort.” Many users find them confusing and difficult to use.
+- **Avoid dependent options.** Avoid making options in one dropdown menu change based on the input to another. Users often don’t understand how selecting an item in one impacts another.
+- **Use a good default.** When most users will (or should) pick a particular option, make it the default: ``<option selected="selected">Default</option>``
```

Comment:
Hows this?
```
**Use a good default.** When most users will (or should) pick a particular option, make it the default. For an uncontrolled component this can be done with the `defaultValue` prop.
```